app/src/main/res/values-es/strings.xml

```diff
@@ -1,98 +1,180 @@
+
+    <string name="illegal_state_no_session_selected_when_preparation_started">Está intentando iniciar la preparación de la sesión sin haber seleccionado una anteriormente.</string>
+    <string name="illegal_state_error_fetching_asset_lists">Ha fallado la recuperación de la lista de recursos.</string>
+    <string name="illegal_state_downloads_did_not_complete_successfully">Las descargas han fallado debido a: %1$s.</string>
+    <string name="illegal_state_failed_to_copy_assets_to_local">Ha fallado la copia de los recursos al almacenamiento local.</string>
+    <string name="illegal_state_failed_to_copy_assets_to_filesystem">Ha fallado la copia de los recursos al sistema de archivos.</string>
+    <string name="illegal_state_failed_to_extract_filesystem">Ha fallado la extracción del sistema de archivos.</string>
+    <string name="illegal_state_filesystem_is_missing_assets">Al sistema de archivos le faltan recursos. Prueba a borrar los archivos de soporte.</string>
+    <string name="illegal_state_failed_to_clear_support_files">Ha fallado la copia de los archivos de soporte.</string>
+
+    <!-- Misc illegal state -->
+    <string name="illegal_state_unhandled_session_service_type">Intentando iniciar una sesión con un tipo de servicio desconocido.</string>
+
+    <!-- Prompts -->
+    <string name="prompt_filesystem">Sistema de Archivos:</string>
+    <string name="prompt_service_type">Tipo de Servicio:</string>
+    <string name="prompt_client_type">Tipo de Cliente:</string>
+    <string name="prompt_app_connection_type_preference">Seleccione el tipo de conexión: </string>
+
+    <!-- Notification strings -->
+    <string name="services_notification_channel_id">ServiciosDeUserLAnd</string>
     <string name="services_notification_channel_name">Servicios</string>
+    <string name="services_notification_channel_description">Servicios persistentes</string>
     <string name="service_notification_title">UserLAnd</string>
-    <string name="services_notification_channel_id">UserLAndServices</string>
-    <string name="service_notification_description">UserLAnd está ejecutando un servicio en segundo plano.</string>
-    <string name="progress_fetching_asset_lists">Obteniendo lista de archivos necesarios…</string>
-    <string name="progress_checking_for_required_updates">Comprobando actualizaciones…</string>
-    <string name="progress_downloading">Descargando archivos necesarios…</string>
-    <string name="progress_starting">Iniciando el servicio…</string>
-    <string name="progress_downloading_out_of">%1$d de %2$d descargados</string>
-    <string name="progress_copying_downloads">Copiando los archivos descargados al almacenamiento local…</string>
+    <string name="service_notification_description">UserLAnd se está ejecutando en segundo plano.</string>
+
+    <!-- Progress bar messages -->
+    <string name="progress_start_step">Dejando todo listo…&&#8230;</string>
```

Comment:
Android Studio mentions there is an error here: 
"Unescaped & or nonterminated character/entity reference"


Here is the English string for reference:
`<string name="progress_start_step">Getting things ready for you&#8230;</string>`


Comparing line 89 and above English string, there seems to be an extra `&` symbol added.  Removing it should allow the project to be build without any hitches.